DigitalOffice/Services/ProjectService/Validators/AddUserToProjectRequestValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using ProjectService.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProjectService.Validators
+{
+    public class AddUserToProjectRequestValidator : AbstractValidator<AddUserToProjectRequest>
+    {
+        public AddUserToProjectRequestValidator()
```

Comment:
Spaces?

Reply:
I will write in more detail. It seems that we will separate the rules with empty lines everywhere